apps/heft/src/plugins/ApiExtractorPlugin/ApiExtractorPlugin.ts

```diff
@@ -64,16 +64,11 @@ export class ApiExtractorPlugin implements IHeftPlugin {
   ): Promise<void> {
     const { heftConfiguration, buildFolder, debugMode, watchMode, production } = options;
```

Comment:
It looks like you converted `api-extractor-task.json` to be riggable in this PR, but  `api-extractor.json` is not yet supported. Is that right?  

My PR for the API Extractor CLI fixes how it processes an external config file path.  Should I fix Heft in my PR as well?

Reply:
>It looks like you converted api-extractor-task.json to be riggable in this PR, but api-extractor.json is not yet supported. Is that right?

Yeah.

>Should I fix Heft in my PR as well?

What's broken in Heft?

Reply:
The api-extractor.json file bases its paths on a `"projectFolder"` setting, whose default value is determined from the file path for **api-extractor.json**.  So if **api-extractor.json** is in a rig project, then API Extractor incorrectly considers the rig project to be the thing that we're analyzing.

PR https://github.com/microsoft/rushstack/pull/2232 will fix this.